tapestry.php

```diff
@@ -187,6 +195,10 @@ function tapestry_enqueue_vue_app()
                 'wpCanEditTapestry' => current_user_can('edit_post', get_the_ID()),
                 'currentUser' => $currentUser,
                 'uploadDirArray' => wp_upload_dir(),
+                'kaltura' => array(
+                    "partnerId" => $kaltura_partner_id,
```

Comment:
Do we want to move `kalturaStatus` here so we don't need to make an API call in the frontend to fetch the status?